src/Modules/PocketWaifu.cs

```diff
@@ -429,18 +429,18 @@ public async Task UseItemOnCardAsync([Summary("nr przedmiotu")] int itemNumber,
         [Command("lazyp")]
         [Alias("lp")]
         [Summary("otwiera pierwszy pakiet z domyślnie ustawionym niszczeniem kc na 2 oraz tagiem wymiana")]
-        [Remarks(""), RequireAnyCommandChannelOrLevel(200)]
-        public async Task OpenPacketLazyModeAsync([Summary("czy zniszczyć karty nie będące na liście życzeń i nie posiadające danej kc?")] uint destroyCards = 2, [Summary("czy zamienić niszczenie na uwalnianie")] bool changeToRelease = false,
-            [Summary("oznacza niezniszczone karty podanym tagiem")] string tag = "wymiana", [Summary("oznacza karty z wishlisty podanym tagiem")] string tagWishlist = "ulubione")
+        [Remarks("2 nie Wymiana Ulubione"), RequireAnyCommandChannelOrLevel(200)]
+        public async Task OpenPacketLazyModeAsync([Summary("czy zniszczyć karty nie będące na liście życzeń i nie posiadające danej kc? (opcjonalne)")] uint destroyCards = 2, [Summary("czy zamienić niszczenie na uwalnianie (opcjonalne)")] bool changeToRelease = false,
```

Comment:
"opcjonalne" is unnecessary

Reply:
All 'opcjonalne' from pack's summary or only this two in #433?

Reply:
all summaries*

Reply:
All summaries in all commands